src/epyq/_build.py

```diff
@@ -1,6 +1,5 @@
 # This file should be overwritten when a distribution build is done
 
-

```

Comment:
Nitpick. This is the only change in the file and it's not necessary. Undo, please.

Reply:
6f19bd2365857e95228cba5bbde21aa4c3f41845